src/yaml/ingestionTrigger/rasterLayerExample.yaml

```diff
@@ -0,0 +1,95 @@
+openapi: "3.0.1"
+info:
+  title: Raster layer examples
+  version: 1.0.0
+components:
+  examples:
+    NewRasterLayerExample:
+      value:
+        metadata:
+          productId: string
+          productName: string
+          productType: Orthophoto
+          productSubType: string
+          description: string
+          srs: string
+          srsName: string
+          transparency: TRANSPARENT
+          region: string
+          classification: string
+          producerName: string
+          scale: 100000000
+        partData:
+          - id: string
+            name: string
+            imagingTimeBeginUTC: "2024-01-28T13:47:43.427Z"
+            imagingTimeEndUTC: "2024-01-28T13:47:43.427Z"
+            resolutionDegree: 0.072
```

Comment:
in all examples change to min of 0.703125

Reply:
done